src/extension/providers/alby/index.ts

```diff
@@ -5,22 +5,13 @@ import WebBTCProvider from "~/extension/providers/webbtc";
 import WebLNProvider from "~/extension/providers/webln";
 
 export default class AlbyProvider extends ProviderBase {
-  liquid: LiquidProvider;
-  nostr: NostrProvider;
-  webbtc: WebBTCProvider;
-  webln: WebLNProvider;
+  webln = new WebLNProvider();
```

Comment:
Can we, instead of creating completely new ones, reference our own objects here so they only exist once? Otherwise `enable()` states are not shared between objects. 

```js
const liquid = new LiquidProvider();
window.liquid = liquid;
...
window.alby = new AlbyProvider(liquid, nostr, webbtc, webln);
```

Reply:
resolved as per discussion in slack